test/src/tests-batchUploader.ts

```diff
@@ -168,8 +167,6 @@ describe('batch uploader', () => {
             pendingEvents[1].EventName.should.equal(10);
             pendingEvents[2].EventName.should.equal('Test Event');
 
-            window.fetchMock.post(urls.eventsV3, 200);
```

Comment:
I moved all references of mocking each events v3 endpoint from each test module to the test-main.js file https://github.com/mParticle/mparticle-web-sdk/pull/313/files#diff-30fa2bc6f63bcb9fbef2dbd45dfd00db46295e495547438dfc80e67ac3f4e5afR22-R27